modules/executable/01_mod.mk

```diff
@@ -57,7 +57,7 @@ define template_for_target
 	$(YQ) 'with(.builds[]; select(.id == "$(1)") | .ldflags[0] = "-s")' | \
 	$(YQ) 'with(.builds[]; select(.id == "$(1)") | .ldflags[1] = "-w")' | \
 	$(YQ) 'with(.builds[]; select(.id == "$(1)") | .ldflags[2] = "$(go_$1_ldflags)")' | \
-	$(YQ) 'with(.builds[]; select(.id == "$(1)") | .gobinary = "$(GO)")' | \
+	$(YQ) 'with(.builds[]; select(.id == "$(1)") | .tool = "$(GO)")' | \
```

Comment:
https://goreleaser.com/deprecations/#buildsgobinary

For example:
 * https://github.com/cert-manager/cmctl/actions/runs/16198123348/job/45729869122
```
  • setting defaults
    • DEPRECATED: builds.gobinary should not be used anymore, check https://goreleaser.com/deprecations#buildsgobinary for more info
```